docs/varnish/vcl/varnish4.vcl

```diff
@@ -167,6 +163,18 @@ sub ez_purge {
     }
 }
 
+sub ez_purge_acl {
+//    if (req.http.x-purge-token) {
```

Comment:
`HTTPCACHE_VARNISH_INVALIDATE_TOKEN_NAME` has default value https://github.com/ezsystems/ezplatform-http-cache/pull/72/files#diff-bf4ea0152820eeb715fdd53160bca64dR39 and can be ommited in meta repo config

Reply:
Yes, but if the header names used is literally always going to `x-purge-token`, are there any need for it to be exported as config parameter and env variable?

Reply:
I think it should be possible to change the token name, but no need to set the same value as default one, I've modified the description for meta repo.

Reply:
I think I agree with @vidarl here, don't see a need for allowing to set different header name.
We don't expose setting to use different method for PURGE for instance.

Also be aware this might be thrown away in the next half year when we get forced to move to FosHttpCache 2 support (for Symfony4), which means lots of logic in this repo should be dropped.

Reply:
done